package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@primer/view-components",
-  "version": "0.46.0",
+  "version": "0.47.0",
```

This isn't necessary.

```suggestion
  "version": "0.46.0",
```